core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1098,6 +1098,15 @@ package object config {
       .booleanConf
       .createWithDefault(true)
 
+  private[spark] val HADOOP_RDD_METADATA_CACHE_SIZE =
```

Comment:
Let's not make this configurable. Just choose some largeish reasonable cap

Reply:
It has been modified, and it feels that 1000 is completely enough.

By the way, the size of a jobconf is 110KB, and 1000 pieces will occupy about 110MB.
do you think it's reasonable？